### PR TITLE
Multiproof for norm equality check

### PR DESCRIPTION
**Pull request by junyechen1996**

### CODE REVIEW

File: poc/vdaf_pine.py [hunk 1]
```diff
@@ -754,6 +767,7 @@ def __init__(self,
             chunk_length_norm_equality=chunk_length_norm_equality,
             num_shares=num_shares,
             num_proofs=2,
+            num_proofs_norm_equality=2,
```
Comment (cjpatton) [inline poc/vdaf_pine.py:770]: What's the motivation for this change? `1` may be a reasonable default given that this part isn't subject to offline attacks.

**Review by cjpatton — APPROVED** [review]
Looks good to me. I'd suggest adding a unit test to exercise multiple squared norm equality proofs, if we go with 1 proof for Field64.

File: poc/vdaf_pine.py (hunk 1 above)
Comment (cjpatton) [reply poc/vdaf_pine.py:770]: That would be better.

File: poc/tests/test_vdaf.py [hunk 2]
```diff
@@ -122,6 +124,61 @@ def test_fields(self):
             self.assertEqual(pine.flp_norm_equality.field, FieldType)
             self.assertEqual(pine.flp.field, FieldType)
 
+    def test_params_combo(self):
+        class PineTest(Pine[Field64, XofTurboShake128]):
```
Comment (cjpatton) [inline poc/tests/test_vdaf.py:128]: nit: I don't think it's necessary to define a sub-class. For this test you could just call `Pine()` I think.

**Review by cjpatton — APPROVED** [review]

**Review by albertpl — APPROVED** [review]

File: poc/tests/test_vdaf.py (hunk 2 above)
Comment (junyechen1996) [reply poc/tests/test_vdaf.py]: Good call. I have to remove the `@abstractmethod` decoration on `__init__` for this to work.

**Pull request merged**